### PR TITLE
Test project refactor

### PR DESCRIPTION
**Pull request by JeffreyDeYoung**

### CODE REVIEW

File: test-helpers/src/test/resources/config/local_test/environment.properties [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+# Cassandra details
+cassandra.contactPoints = 127.0.0.1
+cassandra.port = 9142
+cassandra.keyspace = docussandra
+#cassandra.dataCenter = us-east
+#cassandra.readConsistencyLevel = LOCAL_QUORUM
+#cassandra.writeConsistencyLevel = LOCAL_QUORUM
+
+#Configuration for the MetricsPlugin/Graphite
+metrics.isEnabled = true
```
Comment (marc-hamilton-at-pearson) [inline test-helpers/src/test/resources/config/local_test/environment.properties]: I have no experience with Graphite, so if this is a stupid question, please ignore.

I was just noticing that by default, you have this `metrics.isEnabled` property is set to **true** by default, yet it looks like, based on the rest of the parameters, the person using this test-helper library would have to change some of the default properties to successfully make use of graphite (i.e. the default `metrics.prefix` and `metrics.graphite.host` values).

Should the `metrics.isEnabled` be set to **false** by default instead?

Again, if I'm way off base, please ignore...

**Comment by marc-hamilton-at-pearson** [comment]
Looks like mostly refactoring and housekeeping...  Approved!

File: test-helpers/src/test/resources/config/local_test/environment.properties (hunk 1 above)
Comment (JeffreyDeYoung) [inline test-helpers/src/test/resources/config/local_test/environment.properties]: Valid point; this file is pretty old and has some fields that are no longer valid. I'll clean up.

Comment (JeffreyDeYoung) [inline test-helpers/src/test/resources/config/local_test/environment.properties]: It turns out removing this config is more painful than I expected. I put in a ticket to track it: https://github.com/PearsonEducation/Docussandra/issues/65

**Pull request merged**